pkg/api/types.go

```diff
@@ -1007,6 +1007,12 @@ type PodSecurityContext struct {
 	// Use the host's ipc namespace.
 	// Optional: Default to false.
 	HostIPC bool `json:"hostIPC,omitempty"`
+
+	// SupplementalGroups can be used to specify a list of
+	// additional groups which the main container process will run
+	// as. This will be applied to all containers in the pod in
+	// addition to the primary group of the container.
+	SupplementalGroups []int `json:"supplementalGroups,omitempty"`
```

Comment:
should this be int or uint or uint32 or int64 ?  I think gid_t is unsigned 32, at least on x86.  This probably also applies to other fields.  Sigh...